integration-libs/s4om/.release-it.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
What is the reasoning behind creating s4om as integration library instead of feature library?

Reply:
Hi @szczepaniak-michal,


S4OM library requires integration with S4HANA and Commerce cloud to be setup and configured prior to viewing the schedule lines in the UI. Hence, it was decided long back to add this as an integration-library.

I believe even you were informed of this during the initial developments.

Reply:
Ok thank you